tests/ntp/test_ntp.py

```diff
@@ -10,9 +10,13 @@
 ]
 
 @pytest.fixture(scope="module")
-def setup_ntp(ptfhost, duthost):
+def setup_ntp(ptfhost, duthost, creds):
     """setup ntp client and server"""
 
+    if creds.get('proxy_env'):
+        # If testbed is behaind proxy then force ntpd inside ptf use local time
+        ptfhost.command("sed -i \"/server ntp.your-provider.example/c\server 127.127.1.0 prefer\" /etc/ntp.conf")
```

Comment:
can you use lineinfile module? https://docs.ansible.com/ansible/latest/modules/lineinfile_module.html

Reply:
Changed to lineinfile module